<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceUpdates::CDateTimeSetting</class>
 <widget class="QWidget" name="ServiceUpdates::CDateTimeSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MainMenu::CDateTime" name="widgetDateTime" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenu::CDateTime</class>
   <extends>QWidget</extends>
   <header>MainMenu/Include/DateTime.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
